src/Files.App/UserControls/SideBar/SideBarView.xaml

```diff
@@ -124,33 +124,28 @@
 				UseSystemFocusVisuals="True" />
 		</Border>
 
-		<Grid
+		<!--  Content  -->
+		<ContentPresenter
 			x:Name="ContentPresenter"
 			Grid.Row="1"
 			Grid.Column="2"
 			Margin="2,0,8,0"
-			HorizontalAlignment="Stretch">
-
-			<!--  Content  -->
-			<Border
-				x:Name="Content"
-				VerticalAlignment="Stretch"
-				Child="{x:Bind InnerContent, Mode=OneWay}"
-				TabFocusNavigation="Local" />
+			VerticalAlignment="Stretch"
+			Content="{x:Bind InnerContent, Mode=OneWay}"
+			TabFocusNavigation="Local" />
 
-		</Grid>
 		<VisualStateManager.VisualStateGroups>
+
 			<VisualStateGroup x:Name="DisplayModes">
 				<VisualState x:Name="Expanded" />
 				<VisualState x:Name="Compact">
 					<VisualState.Setters>
-						<Setter Target="DisplayColumn.Width" Value="{StaticResource SidebarCompactOpenPaneLength}" />
+						<Setter Target="PaneColumnDefinition.Width" Value="{StaticResource SidebarCompactOpenPaneLength}" />
 					</VisualState.Setters>
 				</VisualState>
 				<VisualState x:Name="MinimalCollapsed">
 					<VisualState.Setters>
-						<!--  Minimal mode setters  -->
```

Comment:
Aren't the comments helpful?

Reply:
Never understood why coders disregard comments.  Like they are the only ones allowed to understand what their code does, and then they forget what the hell they coded, when they come back to it, a year or two later.

Reply:
> <VisualState x:Name="MinimalCollapsed">

Cant we understand with only this?

Reply:
Rather I added many comments here tho....
And I dont disregard comments rather I add if it is not really helpful, its because that practice is better for us

Reply:
I was also referring to comments like on line 162. `<!--  Specific to collapsed pane  -->`

Reply:
Done

Reply:
Regarding my earlier comment, I was joking with it.